src/org/zaproxy/zap/extension/advreport/resources/help/contents/customreport.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>
+Customizable HTML Report
```

Comment:
The "Customizable HTML Report" could be changed to "Custom Report" (to normalise with the name of the add-on).